radis/test/lbl/test_loader.py

```diff
@@ -222,13 +222,179 @@ def test_custom_abundance(verbose=True, plot=False, *args, **kwargs):
     assert s.compare_with((3 * s3.take("abscoeff")), "abscoeff", rtol=0.5e-2, plot=True)
 
 
+# Added in https://github.com/radis/radis/pull/580
+# Compares dataframes fetched and manipulated with vaex with that of pandas. Every column of dataframe in vaex format is compared with corresponding column of dataframe in pandas format .
+def test_vaex_and_pandas_dataframe_fetch_databank():
+    import numpy as np
+
+    from radis.lbl import SpectrumFactory
+
+    sf = SpectrumFactory(
+        1500,
+        2000,
+        molecule="CO",
+        isotope="1,2,3",
+        pressure=1.01325,  # bar
+        path_length=1,  # cm
+    )
+
+    def compare_dataframe(df_vaex, df_pandas, columns):
+        if_same = True
+        for column in columns:
+            if_same = if_same * np.all(df_vaex[column].to_numpy() == df_pandas[column])
+
+        return if_same
+
+    # HITRAN
+    # Comparing Vaex dataframe and Pandas dataframe fetched from HITRAN
+
+    # Fetching in vaex dataframe format
+    sf.dataframe_type = "vaex"
+    sf.fetch_databank(
+        "hitran", memory_mapping_engine="vaex", output="vaex", load_columns="all"
+    )
+    df1 = sf.df0
```

Comment:
assert that it's the right output
i.e. something like 
```
assert isinstance(df1, vaex.DataFrame)
```

Same idea for df2 below